.travis.yml

```diff
@@ -7,7 +7,11 @@ sudo: false
 script:
   - cargo build --verbose
   - cargo test --verbose
-  - [ "$TRAVIS_RUST_VERSION" = "1.0" ] || cargo test --verbose --features no_std
+  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
+       cargo test --verbose --features no_std
```

Comment:
Is there a reason, why we test against 1.0 and not against stable? When that's answered I'll amend the PR.